checker-plugin/src/main/scala/org/virtuslab/akkasaferserializer/AkkaSerializabilityCheckerPluginComponent.scala

```diff
@@ -46,10 +48,10 @@ class AkkaSerializabilityCheckerPluginComponent(val pluginOptions: PluginOptions
                 case None =>
                   reporter.error(
                     next.typeSymbol.pos,
-                    s"""${next.toString()} is used as Akka message but does not extend a trait annotated with ${classOf[
```

Comment:
Oh ok... the error was raised here right?

Reply:
Exactly